code/modules/clothing/glasses/glasses.dm

```diff
@@ -652,11 +663,18 @@ var/list/science_goggles_wearers = list()
 	update_emitter()
 
 /obj/item/clothing/glasses/emitter/proc/update_emitter()
-	if (!emitter || !isturf(emitter.loc) || emitter.lying)
+	if (!emitter || !isturf(emitter.loc))
 		if (beam)
 			qdel(beam)
 			beam = null
 		return
+	if (ismob(emitter) && emitter:lying)
```

Comment:
Looks like you accidentally left the : operator here, which is redundant as it gets checked below anyways

Reply:
yep